go/vt/vttablet/tabletmanager/rpc_replication.go

```diff
@@ -837,6 +837,11 @@ func (tm *TabletManager) PromoteReplica(ctx context.Context, semiSync bool) (str
 	}
 	defer tm.unlock()
 
+	tm.replManager.SetTabletType(topodatapb.TabletType_PRIMARY)
```

Comment:
Can you add a comment here that this doesn't change the tablet's type, it only stops/starts replManager ticks?
Let us also add a comment on the function definition itself.
Otherwise LGTM

Reply:
Fixed. As discussed, I have also added the comments for `wrangler.SetReplicationSource` explaining that we ignore the error in finding the shard primary from the topo server.